src/prelaunch_data_generator/el_cl_genesis/el_cl_genesis_generator.star

```diff
@@ -46,29 +26,9 @@ def generate_el_cl_genesis_data(
 
     template_data = new_env_file_for_el_cl_genesis_data(
         genesis_unix_timestamp,
-        network_id,
-        deposit_contract_address,
-        seconds_per_slot,
-        preregistered_validator_keys_mnemonic,
         total_num_validator_keys_to_preregister,
-        genesis_delay,
-        max_per_epoch_activation_churn_limit,
-        churn_limit_quotient,
-        ejection_balance,
```

Comment:
removing all of this?

Reply:
little cleanup. was a messy set, now I just pass network_params.